image_to_table/image_with_word_boxes.py

```diff
@@ -1,146 +1,87 @@
 import io
+from typing import List, Iterable
+import bisect
 
+import numpy as np
 import cv2
-import matplotlib.pyplot as plt
 from google.cloud import vision
 
-from image_to_table.models import Box, Point
+from image_to_table.models import TextBox
+from opencv_wrapper import Rect
 
 
-def detect_text(filename):
+def create_text_box(text) -> TextBox:
+    tl, tr, br, bl = text.bounding_poly.vertices
```

Comment:
these variable names are a bit cryptic too me

Reply:
It's the way OpenCV names the corners; top-left, top-right, bottom-right, bottom-left. In c++ they have a `Rect` class with properties (actually member functions) `tl()` and `br()`. So in `opencv_wrapper` I have properties `tl`, `tr`, `br` and `bl`. It is a bit cryptic, but at the same time `top_left` etc. is very verbose when using it actively.

But I'm not completely rigid on this.

Reply:
Ah ok! true, it does become very verbose. I'm fine with leaving it as is, I just didn't realize what they stood for - but they make sense!